grafeas-charts/values.yaml

```diff
@@ -36,3 +36,11 @@ certificates:
       -----BEGIN RSA PRIVATE KEY-----
       ...
       -----END RSA PRIVATE KEY-----
+
```

Comment:
Should this contain the values for readiness and liveness probes?

Reply:
I've added both as default `{}` and examples of how to define them if they're required.

Reply:
I've just updated the values based on a few tests watching the pod logs after deployment. Seems like ~12 secs startup on average so have set initial delay to 15 secs.

Reply:
Ok great! Let's add this in a comment, for posterity.

Reply:
Sure

Reply:
Ok, I've now added `failureThreshold` and some comments on the best/worst case startup times taken into account for the default values.